### DISCUSSION TITLE
Welcome to Discussions of micro-app

### DISCUSSION THREAD

**Discussion by bailicangdu** [jd-opensource/micro-app#6] [Announcements]
1

**Comment by 513205785Xie** [comment]
你好 我本地下载了[micro-app](https://github.com/micro-zoe/micro-app)的代码后  本地想用build这个命令把micro-app这个lib打起来  因为我看到其他的子项目有用这个本地打的包  但是我运行build命令一直会报错  解决到了一个我无从下手的错误后就卡住了。 请问可以麻烦您写一份具体的本地打包项目包的步骤吗？   我本地是win 10的环境。 谢谢啦

**Reply by bailicangdu** [reply]
我的问题，没有添加cross-env做window的适配，你重新拉一下代码

**Reply by 513205785Xie** [reply]
ok  我试一下   顺便问一下  有讨论群什么的吗  我想找组织

**Reply by bailicangdu** [reply]
推荐使用gitter，https://gitter.im/zoe-community/zoe-room
